<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spell_and_cantrip_display</class>
 <widget class="QWidget" name="spell_and_cantrip_display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>-20</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Spells and Cantrips</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>109</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6"/>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>750</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10"/>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>610</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>109</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9"/>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>109</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>109</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4"/>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_16">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>109</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7"/>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_15">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>109</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>750</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_14">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5"/>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_13">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>470</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>109</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8"/>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>261</width>
     <height>781</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>259</width>
      <height>779</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="description_text">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>781</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantrips</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_cantrip">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>219</width>
      <height>109</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>730</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_7">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>590</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>590</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>730</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>730</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Info</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Spell casting ability</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Spell save dc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Spell attack bonus</string>
    </property>
   </widget>
   <widget class="QLabel" name="spell_cast_ability">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spell_save_dc">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spell_attack_bonus">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
